suggestions/dialers.json

```diff
@@ -1 +1,16 @@
-[]
\ No newline at end of file
+[
+    {
+        "id": "com.simplemobiletools.dialer",
+        "label": "Simple Dialer",
+        "source": "fg",
+        "reason": "Manage phone calls with this simple phone dialer app with phonebook",
+        "repo": "https://github.com/SimpleMobileTools/Simple-Dialer"
+    },
+    {
+        "id": "com.chooloo.www.koler",
+        "label": "Koler",
+        "source": "f",
+        "reason": "Uniquely stylized phone app with customizable features",
+        "repo": "https://github.com/Chooloo/koler"
+    }
```

Comment:
Require reviews. @shuvashish76: Any opinions?

Reply:
AFAIK, Simple Dialer is the **only dialer app** which is able to disable proximity sensor, and that was a game-changer for one of my smartphones which has a broken proximity sensor!